File: Data Structures & Algorithms/missing-number/submission-0.cs
public class Solution {
    public int MissingNumber(int[] nums) {
        var res = 0;
        var n = nums.Length;

        for(var i = 0; i < n; i++)
        {
            res ^= i;
            res ^= nums[i];
        }

        res ^= n;

        return res;

    }
}

File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    # brute force
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        res = [0 for i in range(len(temperatures))]

        for i in range(len(temperatures)):
            for j in range(i, len(temperatures)):
                if temperatures[j] > temperatures[i]:
                    res[i] = j - i
                    break
        return res
